Add sd-webui-civbrowser.json

SignalFlagZ opened the pull request:
## Info 
<!--- Repo url or any other thing you like to say --->
https://github.com/SignalFlagZ/sd-webui-civbrowser
Extension to search and download Civitai models in multiple tabs. Save model information. Send sample infotext to txt2img.
## Checklist:
<!--- Checkboxes will become clickable after submit, no need to fill them now --->
- [x] I have read the [`Readme.md`](https://github.com/AUTOMATIC1111/stable-diffusion-webui-extensions)
- [x] The description is written in English.
- [x] The `index.json` and `extension_template.json` have not been modified.
- [x] The `entry` is placed in the `extensions` directory with the `.json` file extension.

w-e-w commented:
reusing the same reply https://github.com/AUTOMATIC1111/stable-diffusion-webui-extensions/pull/180#issuecomment-1692388830 can you answer these questions
if the situation has not changed since then I will just add your extension and assume that the old one is deprecated
(without asking you to change the repo name)

SignalFlagZ commented:
Thank you for maintaining and managing the extensions.
My repository is very different from the original because I changed the object-oriented code to achieve multi-tabbing. Therefore, I am not considering merging it with the original. Also, I have already renamed my repository. I prefix the element ID and class name with `civsfz` to make them unique.

w-e-w commented:
> I have already renamed my repository

apparently the health check I did earlier today missed the fact that I'm blind 🫠
